backend/agentd/session.go

```diff
@@ -261,7 +262,10 @@ func (s *Session) Stop() {
 	close(s.checkChannel)
 	for _, sub := range s.cfg.Subscriptions {
 		ring := s.ringPool.Get(ringv2.Path(s.cfg.Namespace, sub))
-		logger.Info("removing agent from ring")
+		logger.WithFields(logrus.Fields{
+			"namespace": s.cfg.Namespace,
+			"id":        s.cfg.AgentName,
```

Comment:
Maybe use the key "agent" instead of "id", just to remove any ambiguity about what the name is?